src/shared/Core/CredentialStore.cs

```diff
@@ -66,7 +79,7 @@ private void EnsureBackingStore()
             {
                 case StoreNames.WindowsCredentialManager:
                     ValidateWindowsCredentialManager();
-                    _backingStore = new WindowsCredentialManager(ns);
+                        _backingStore = new WindowsCredentialManager(ns);
```

Comment:
Nit: I think we can remove this.

Reply:
$ git branch -m master main
$ git fetch origin
$ git branch -u origin/main main